docker-bench-security.sh

```diff
@@ -19,6 +19,7 @@ dir_name=`dirname ${this_path}`    ## Dir where this file is
 myname=`basename ${this_path}`     ## file name of this script.
 logger="${myname}.log"
 
+export PATH=/bin:/sbin:/usr/bin
```

Comment:
You mac users you. ;)

Absolutely, the PATH change is just there to add some sort of basic limitation. As /usr/local/bin is required, it should be added.